website/source/api/system/plugins-catalog.html.md

```diff
@@ -9,17 +9,17 @@ description: |-
 
 # `/sys/plugins/catalog`
 
-The `/sys/plugins/catalog` endpoint is used to list, register, update, and
+The `/sys/plugins/catalog` endpoint is used to read, register, update, and
 remove plugins in Vault's catalog. Plugins must be registered before use, and
 once registered backends can use the plugin by querying the catalog.
 
-## List Plugins
+## Read Plugins
```

Comment:
Even though the Method has changed to GET, I think "List" is still the right word for the heading, and is consistent with following sentence.

Reply:
Good point! Also, revisiting this made me realize I'd not included the new three LIST endpoints for plugins by type, so I'm adding those.